de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/console/ShareCommand.java

```diff
@@ -0,0 +1,54 @@
+package de.fu_berlin.inf.dpp.server.console;
+
+import de.fu_berlin.inf.dpp.filesystem.IWorkspace;
+import de.fu_berlin.inf.dpp.server.filesystem.ServerProjectImpl;
+import de.fu_berlin.inf.dpp.session.ISarosSession;
+import de.fu_berlin.inf.dpp.session.ISarosSessionManager;
+import de.fu_berlin.inf.dpp.session.User;
+import java.io.PrintStream;
+import java.util.*;
```

Comment:
It's weird that the formatter does not have a problem with this. According to the google style, wildcard imports are never used. Please remove the wildcard import.

I thought this was already reflected by the current formatting rules in Intellij and Eclipse. How did you format the code?

Reply:
I have just used the standalone formatter, but that should fix that as well, right?

Reply:
Fixed it manually.